### ISSUE TITLE
Installing gets Stuck

### ISSUE THREAD

**Issue by M0WW0M** [DotBow/Blender-Launcher#88]
**Describe the bug**
When downloading specific experimental Versions the installation gets stuck while unzipping

**To Reproduce**
1. Go to Experimemtal
2. Try downloading Porting Modifers ... as seen in the image

**Screenshots**
![grafik](https://user-images.githubusercontent.com/55852814/129483140-4bb37f14-f289-4cfa-88c6-8fb0593fabab.png)
one Example even after 2h there is no change

**System (please complete the following information):**
 - Operation system:  Windows 10
 - Blender Launcher version: v.1.13.0

**Comment by DotBow** [comment]
Hi!
Is there a log file near by .exe?

**Comment by M0WW0M** [comment]
There was! Thanks for the quick respons
[log_zipProblem.txt](https://github.com/DotBow/Blender-Launcher/files/6990309/log_zipProblem.txt)

**Comment by DotBow** [comment]
> 
> 
> There was! Thanks for the quick respons
> [log_zipProblem.txt](https://github.com/DotBow/Blender-Launcher/files/6990309/log_zipProblem.txt)

Thanks!
Can't reproduce on my side, builds downloads and unpacks correctly. Need to go deeper on this one..

**Comment by M0WW0M** [comment]
If you need any more information on System or other stuff to help fixing this bug, feel free to ask!

**Comment by DotBow** [comment]
> 
> 
> If you need any more information on System or other stuff to help fixing this bug, feel free to ask!

For now I can suggest you to delete exiting bad builds from library and try again.
If it will not help then try to run latest dev version.

https://drive.google.com/file/d/1Y6JbTZl2i1dof26jUiMLrLjYm8r9Ku_1/view?usp=sharing

**Comment by M0WW0M** [comment]
![grafik](https://user-images.githubusercontent.com/55852814/129873127-8dba7244-d3ff-4803-a381-6611eaa824ca.png)
[Blender Launcher.log](https://github.com/DotBow/Blender-Launcher/files/7006079/Blender.Launcher.log)

Sadly even in the dev Version the Problem is present.
and the bad builds never arrive in my library, because they never finish the unzipping process.
But thanks for the quick and detailed support.

**Comment by DotBow** [comment]
You say that others loads perfectly fine?

**Comment by M0WW0M** [comment]
Yes for some builds the download works fine, but others get stuck, but i haven't found a pattern in which Versions work and which don't

**Comment by M0WW0M** [comment]
https://user-images.githubusercontent.com/55852814/129902888-15abcb0e-d657-4bc5-8d74-ca568ec41216.mp4

**Comment by DotBow** [comment]
@M0WW0M OK, I have one thought: it might be related to max path length on Windows which is capped to 256 characters. Try to disable max path limit on your system or move Library to other folder (like D:/Blender).

**Comment by M0WW0M** [comment]
Perfekt! Thank you so much that fixed the Problem. Amazing support :D

**Comment by cg-Kdaf** [comment]
Hello, same problem for me here on linux. Using the release version, I can not download the experimental branches. I explored what was happening, and it downloads the hash, to check, but not the blender app. If you want any infos, feel free to ask.

**Comment by cg-Kdaf** [comment]
Logs are :
`2021-08-21 10:37:41,830 - Linux - Blender Launcher 1.13.0
Traceback (most recent call last):
  File "<string>", line 1, in <module>
OSError: [Errno 5] Input/output error
`
It may not be related

**Comment by cg-Kdaf** [comment]
It works for the Cycles X branch, but not the long name called branches.

**Comment by DotBow** [comment]
> Hello, same problem for me here on linux. Using the release version, I can not download the experimental branches. I explored what was happening, and it downloads the hash, to check, but not the blender app. If you want any infos, feel free to ask.

Hi! Thanks for report, but this is totally unrelated issue. Please, create a separate ticket for it.
